dateutil/tz/tz.py

```diff
@@ -941,6 +941,13 @@ def __init__(self, stdabbr, stdoffset=None,
         self._std_abbr = stdabbr
         self._dst_abbr = dstabbr
 
+        if isinstance(stdoffset, datetime.timedelta) and \
```

Comment:
Won't this still allow things like `tz.tzrange(None, 90000)`?

Reply:
when looking at the doc string, i see the spec says:
```python
"""
...
:param stdoffset:
        An integer or :class:`datetime.timedelta` object or equivalent
        specifying the base offset from UTC.
...
"""
```

What would be an equivalent? Also, while its not explicity said, im assuming that if its an int that is provided, its the number of seconds as the offset?

Reply:
Yes, `int` would be seconds. An equivalent would be anything that implements `timedelta`'s interface, I guess.